### ISSUE TITLE
Rule css:S4662 false positive in razor *.cshtml files

### ISSUE THREAD

**Issue by olsh** [SonarSource/sonar-css#249]
In Razor for escaping `@` symbol is used
https://docs.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-3.1#razor-syntax

So in Razor, media query should be escaped like 
```css
@@media only screen {
...
```

Rule css:S4662 triggers on such code, but it's pretty valid in Razor.
![image](https://user-images.githubusercontent.com/3613592/72601364-5375f000-3926-11ea-9f0d-c89d478da28a.png)

**Comment by vilchik-elena** [comment]
Indeed rule can be improved.
As a workaround you can configure the rule in your profile to ignore `@@media`
